docs/publish/centralized-deployment.md

```diff
@@ -114,7 +113,7 @@ After an Office Add-in has been published via Centralized Deployment, any change
 
 - **Line-of-business add-in**: If an admin explicitly uploaded a manifest file when implementing Centralized Deployment via the Office 365 admin center, the admin must upload a new manifest file that contains the desired changes. After the updated manifest file has been uploaded, the next time the relevant Office applications start, the add-in will update.
 
-- **AppSource add-in**: If an admin selected an add-in from AppSource when implementing Centralized Deployment via the Office 365 admin center, and the add-in updates in AppSource, the add-in will update later via Centralized Deployment. The next time the relevant Office applications start, the add-in will update.
+- **Office Store add-in**: If an admin selected an add-in from Office Store when implementing Centralized Deployment via the Office 365 admin center, and the add-in updates in Office Store, the add-in will update later via Centralized Deployment. The next time the relevant Office applications start, the add-in will update.
```

Comment:
Add the word "the":
- change "from Office Store" to "from the Office Store"  
- change "in Office Store" to "in the Office Store"